benchmarks/_benchmark.py

```diff
@@ -116,8 +189,25 @@ def record(
             github=self.github_info,
             options=options,
             output=output,
+            publish=publish,
+        )
+
+        res = BenchmarkResult(
+            name=benchmark["tags"]["name"],
+            run_id=benchmark.get("run_id"),
+            batch_id=benchmark.get("batch_id"),
+            result=benchmark.get("stats"),
+            params=dict(zip(self.valid_cases[0], case)) if case else dict(),
+            tags=benchmark.get("tags"),
+            info=benchmark.get("info"),
+            context=benchmark.get("context"),
+            github=benchmark.get("github"),
+            options=benchmark.get("options"),
+            output=output,
+            error=benchmark.get("error"),
         )
-        return benchmark, output
+
+        return res
```

Comment:
This will not work. Conbench runner needs benchmark, output to be returned by benchmark method.